src/main/java/com/thoughtworks/aceleradora/dominio/AppQuestionario.java

```diff
@@ -0,0 +1,57 @@
+package com.thoughtworks.aceleradora.dominio;
+
+import java.util.Scanner;
+
+import static com.thoughtworks.aceleradora.dominio.Questionario.criaDiagnostico;
+
+public class AppQuestionario {
+    public static void main(String[] args) {
+        Diagnostico novoDiagnostico = criaDiagnostico();
+        Scanner entrada = new Scanner(System.in);
+
+        System.out.println("Olá, boas-vindas! Somos a EasyBeasy.");
+
+        for (Nivel nivel : novoDiagnostico.getNiveis()) {
```

Comment:
Esse código ficou bem complexo. Cinco loopings aninhados. Se a gente for investir mais tempo neste caminho, eu não seguiria adiante antes de tentarmos simplificar essa parte.

Reply:
Como poderiamos simplificar o código?

Reply:
separar em métodos?

Reply:
Um método com todos os `for`?